src/lib/services/access-service.ts

```diff
@@ -643,6 +649,17 @@ export class AccessService {
                 );
             }
         }
+        const addedPermissions = await this.store.getPermissionsForRole(
+            newRole.id,
+        );
+        this.eventService.storeEvent({
+            type: ROLE_CREATED,
+            createdBy: role.createdBy || 'unknown',
```

Comment:
I see potential for error when we have the same default logic in many places.

Reply:
Yes, but this logic should go away once we fix enterprise repo